app/Repositories/Summit/DoctrineSummitEventRepository.php

```diff
@@ -842,6 +842,27 @@ public function getLastPresentationOrderBySummit(int $summit_id): int
         }
     }
 
+    /**
+     * feature/session_overflow
+     * @param string $enc_key
+     * @return SummitEvent|null
+     */
+    public function getByOverflowStreamKey(string $enc_key): ?SummitEvent
```

Comment:
change enc_key to something more meaning full

Reply:
done